index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav id="navbar">
+        <ul>
+          <li>
+            <a href="">About</a>
+          </li>
+          <li>
+            <a href="#projects">Work</a>
+          </li>
+          <li>
+            <a href="#contact">contact</a>
+          </li>
+        </ul>
+      </nav>
+      <section  id="welcome-section">
+        <h1>Hey I am Roshan<br>
+          a web developer
+        </h1>
+      </section>
+      <section id="projects">
+        <div class="project-tile">
+          <a href= "#">These are some of my projects</a>
+        </div>
+        <div class="photo">
+          <img src="1.jpg" alt="image">
+          <img src="2.png" alt="image">
+          <img src="3.png" alt="image">
+        </div>
+        
+      </section>
+      <div class="button">
+        <button class="btn">Show all ></button>
+      </div>
+      <div class="ros">
+        <p>Let's work together...</p>
+      </div>
+      <p id="dip">How do you take coffee?</p>
```

Comment:
- [ ] Always try to use descriptive class names.

Reply:
This one sounds more like a suggestion to me but remember to add a "why" (why the given suggestion is made). I mean remember to explain the reason for the required change.